build.gradle

```diff
@@ -22,13 +22,13 @@ allprojects {
     repositories {
         google()
         jcenter()
-        maven { url "https://jitpack.io" }
+
     }
 }
 
 task clean(type: Delete) {
```

Comment:
Why have you moved this file? And made changes to this.

Reply:
please allow me sometime to check.